rfcs/0005-single-cluster-dnspolicy.md

```diff
@@ -196,16 +196,15 @@ DNSRecord:
 
 ManagedZone:
 
-- `spec.dnsProviderSecretRef` replaced with `spec.providerRef`
-- new api version `v1alpha2`
+- ManagedZone API wil be removed and no longer supported as part of MGC/Kuadrant.
 
 ### DNSPolicy.spec.providerRef
 
-The `providerRef` field is mandatory and contains a reference to a resource that shows how DNSRecords will be reconciled.
-    - `spec.providerRef.name` - name of the provider resource
-    - `spec.providerRef.kind` - kind of resource, can be anything but only `Secret` or `Managedzone` will be reconciled by the `DNSPolicy` controller.
+The `providerRef` field is mandatory and contains a reference to a secret containing provider credentials.
+    - `spec.providerRef.name` - name of the provider resource.
+    - `spec.providerRef.kind` - kind of resource, must be `Secret`.
```

Comment:
Any reason to keep this field then?

Reply:
Yeah, i did think about that. We probably can just remove the kind and expect a secret name since we are saying now you must always set this and it must be a secret.

Reply:
yep, sounds good to me!

Reply:
removed